pre_commit/resources/hook-tmpl

```diff
@@ -40,6 +41,7 @@ fi
 {pre_push}
 
 # Run pre-commit
+args="$args --hook-stage $hook_stage"
```

Comment:
Yeah `pre-commit` alone defaults to `pre-commit run`.  Mostly to make the normal case *just work*.  pre-push was added later.